tagstudio/src/qt/platform_strings.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2024 Travis Abendshien (CyanVoxel).
+# Licensed under the GPL-3.0 License.
+# Created for TagStudio: https://github.com/CyanVoxel/TagStudio
+
+"""A collection of platform-dependant strings."""
+
+import platform
+
+
+class PlatformStrings:
+    open_file_str: str = "Open in file explorer"
+
+    if platform.system() == "Windows":
+        open_file_str = "Open in Explorer"
+    elif platform.system() == "Darwin":
+        open_file_str = "Reveal in Finder"
```

Comment:
Having the logic directly in the class isnt the usual way how to do it, but I think we can live with it here :thinking:

Reply:
For future reference what would be the expected way to do something like this? I was mulling over different implementations and just went with this since it minimizes the performance impact of the checks while keeping the imports somewhat clean

Reply:
I have to board plane now, so just very quickly - I would probably go slightly different way - rather than doing everything in one class, I'd create a class with common phrases, and then inherit from that class like `WindowStrings` where platform-specific phrases would be overriden.

```
class SharedStrings:
   open_file_str: str = 'something'


class WindowsStrings(SharedStrings):
    open_file_str: str = 'windows something'  

TRANSLATIONS = # `platform.system()` logic to decide which class to use

```